tests/integration/utils/got.cjs

```diff
@@ -20,19 +10,6 @@ const extendedGot = got.extend({
     statusCodes: STATUS_CODE,
   },
   timeout: TIMEOUT,
-  // TODO: remove when https://github.com/sindresorhus/got/issues/1489 is fixed
```

Comment:
Note here: Issue was resolved in https://github.com/sindresorhus/got/pull/2187 so this workaround suggested in https://github.com/sindresorhus/got/issues/1489#issuecomment-805485731 is no longer needed.